supporting_content/dimensions/skupriceid.md

```diff
@@ -0,0 +1,22 @@
+## Example provider mappings 
+
+Current column mappings found in available data sets:
+
+| **Provider** | **Data set**                | **Column**                             |
```

Comment:
nit: formatting. No need for ** to be added here or below in column headings.

Reply:
Revised.